lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -48,6 +48,22 @@ def authorize(money, payment, options = {})
         commit('/pts/v2/payments/', post)
       end
 
+      def refund(money, options = {})
+        post = build_refund_request(money, options)
+
+        yield post if block_given?
```

Comment:
👍

Reply:
Yes, I was wrong copy / paste this approach and it is only necessary for the purchase/authorization methods